modules/ROOT/pages/key-concepts.adoc

```diff
@@ -1,20 +1,20 @@
 = Bonita key concepts
 :description: Learn some of the key concepts related to designing an application or a process with Bonita, and the terminology used in Bonita Studio.
 
-Learn some of the key concepts related to designing an application or a process with Bonita, and the terminology used in Bonita Studio.
+{description}
 
 == Applications and processes
 
 An application is a collection of processes, user interfaces (called pages) and shared data. A page that is a user interface to a process is called a form.
```

Comment:
A page that is a user interface to a process is called a form. => The user interface page of a process is called a form. 

I had troubles understanding the meaning of the phrase. The change aint mandatory because not related to Portal, but goes with the continuos improvment mindset.

Reply:
I added it.